blueprints/ember-bootstrap/wizard.js

```diff
@@ -0,0 +1,162 @@
+const chalk = require('chalk');
```

Comment:
This should also have `'use strict';`!

Some day I changed the Travis setup to node 6 because of some failing upstream dependencies, but I should probably try again to revert to node 4. Though it's reaching end-of-life, cutting support for it would probably require a major version bump, which I din't really want to do yet...